internal/tasks/tasks_test.go

```
@@ -469,6 +469,125 @@ func TestValidatePayload(t *testing.T) {
+tasks:
+  test:
+    script: test.sh
+    input:
+      - name: instance
+        type: string
+        values_url: "http://example.com/values"
+        values_cache: 30s
+`
+		tmpDir := t.TempDir()
+		tmpFile := filepath.Join(tmpDir, "tasks.yaml")
+		os.WriteFile(tmpFile, []byte(content), 0644)
```

golangci-lint (2.12.1)

[error] Error return value of `os.WriteFile` is not checked

(errcheck)